CandidateReranking.md

```diff
@@ -74,27 +72,27 @@ Entites returned from the SemanticSearchAPI.
 
 | Param    | Type | Description
 |----------|------|--------|
-| name | String | **Required**: Name of the keyword, e.g. "Java".
-| type | String | **Required**: Type of the keyword, e.g. "skill".
-| weight | Integer | **Required**: Weight of the keyword as an int.
+| name | String | **Optional**: Name of the keyword, e.g. "Java".
+| type | String | **Optional**: Type of the keyword, e.g. "skill".
+| weight | Integer | **Optional**: Weight of the keyword as an int.
 | relationships | Map<String, Entities[]> | **Optional**: Map of Strings(Semantic Enrichment Type) to a list of of entities. Valid list of Map key values are: `job_titles`, `occupations`, `skills`, `job_level`, `text_kernel_related_keywords`, `interesting_keywords`, `related_keywords`, `related_keywords_recruiter`, `related_keywords_jobseeker`, `raw_job_titles_jobseeker`, `raw_job_titles_recruiter`, `extracted_kewords`, `custom_keywords`, `related_search_terms`, and `job_titles_recruiter`.
 
 #### Entity
 | Param    | Type | Description
 |----------|------|--------|
-| name | String | **Required**: Name of enrichment **Entity**.
-| weight | double | **Reqired**: Weight of enriched term according to the SemanticSearchAPI.
+| name | String | **Optional**: Name of enrichment **Entity**.
```

Comment:
why is Entity capitalized here? also, an example might be good (I'm not personally sure off the top of my head what an "entity" name might be)